python/utilities/ev_whitelist/generate_ev_whitelist.py

```diff
@@ -66,7 +68,8 @@ def _write_cert_and_chain(
              "wb"))
 
 def _ev_match(
-        output_dir, certificate, entry_type, extra_data, certificate_index):
+        output_dir, last_entry_index, certificate, entry_type, extra_data,
```

Comment:
you might use the word 'acceptable' in last_entry_index here too, to be consistent with above, and to make the code better self-documented.